src/main/groovy/com/github/eerohele/SaxonXsltTask.groovy

```diff
@@ -26,22 +31,29 @@ class SaxonXsltTask extends DefaultTask {
     protected static final String PERIOD = '.'
     protected static final String XSLT_NAMESPACE = 'http://www.w3.org/1999/XSL/Transform'
 
-    protected final List<String> defaultArguments = ['-quit:off'].asImmutable()
+    protected static final List<String> DEFAULT_ARGUMENTS = ['-quit:off'].asImmutable()
 
-    protected final Map<String, Object> options = [output: project.buildDir]
+    private Object inputFiles
```

Comment:
Is it necessary to move these things into variables for Gradle 7.1.1? I'd rather not do it if we don't have to. For one thing, this change is in conflict with the changes @ndw proposes in https://github.com/eerohele/saxon-gradle/pull/26.

Reply:
the problem with having these properties inside the Map is that the map itself contains a bunch of xlt-inputs that influence the outputs and gradle needs to know about them in order to perform a reasonable up-to-date check. Therefor the map itself has to be promoted as `@Input`. But when doing this marking individual keys as for #29 will not be possible (or at least I didn't see a way to do it).